llarp/link/link_manager.cpp

```diff
@@ -1062,14 +1062,178 @@ namespace llarp
   void
   LinkManager::handle_path_build(oxen::quic::message m)
   {
+    if (!router.path_context().AllowingTransit())
+    {
+      log::warning("got path build request when not permitting transit");
```

Comment:
All these log statements without `link_cat` are going to error amigo :smile: 
Will fix the log statements at least in my next commit, leaving the rest untouched